packages/block-editor/src/hooks/use-typography-props.js

```diff
@@ -18,12 +19,26 @@ import { getFontSizeClass } from '../components/font-sizes';
  * Provides the CSS class names and inline styles for a block's typography support
  * attributes.
  *
- * @param {Object} attributes Block attributes.
+ * @param {Object}  attributes            Block attributes.
+ * @param {boolean} isFluidFontSizeActive Whether the function should try to convert font sizes to fluid values.
  *
  * @return {Object} Typography block support derived CSS classes & styles.
  */
-export function getTypographyClassesAndStyles( attributes ) {
-	const typographyStyles = attributes?.style?.typography || {};
+export function getTypographyClassesAndStyles(
+	attributes,
+	isFluidFontSizeActive
```

Comment:
Looking at `use-color-props.js` in the same directory as this file, it seems the other files have a function like `getColorClassesAndStyles( attributes )` signature, and then provide a hook (`useColorProps`) that is responsible for doing its own settings look-up.

In a follow-up, we might also look to adding a hook like that to `use-typography-props.js` so that we don't need to do the check for fluid support in the Search block's edit function.

However, even if we were to do that, we'd still need to tell `getTypographyClassesAndStyles` about the state of fluid typography, so we'd still need the change to this signature in that case, I think.

All of which is a long way to say, the change to this function signature sounds good to me!

Reply:
Cheers for thinking about this. I will add a note to follow up on this if we don't decide to change it.